pyproject.toml

```diff
@@ -1,24 +1,46 @@
 [build-system]
-requires = ["setuptools>=42", "wheel"]
+requires = ["setuptools >= 77.0.3"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "patterns"
+name = "python-patterns"
 description = "A collection of design patterns and idioms in Python."
 version = "0.1.0"
 readme = "README.md"
 requires-python = ">=3.9"
-license = {text = "MIT"}
```

Comment:
No license in the project - having a license here will only cause confusion. 

Unless this project should be licensed. In which case a LICENSE.md should be added.